src/client/renderingengine.cpp

```diff
@@ -249,8 +249,10 @@ void RenderingEngine::draw_load_screen(const std::wstring &text,
 #ifndef __ANDROID__
```

Comment:
maybe we can get rid of the android codeblock then?
I can't really tell what it even does.

Reply:
Removing the special case for Android results in a noticeably smaller progress bar on my phone:

before
![screenshot before](https://github.com/minetest/minetest/assets/82708541/f3836fb4-2509-41b0-bc5b-652eae9a8dbb)

after
![screenshot after](https://github.com/minetest/minetest/assets/82708541/a02d322c-092f-40fc-9964-794ee8bc075e)

It looks better with the special case, I'd say.

Reply:
not that much of a difference IMO and I'd rather get rid of the weird hack

Reply:
Alright, but please in another PR so that this one can stay a trivial bugfix.